### PR TITLE
Add test coverage for all importers in examples

### PR DESCRIPTION
**Pull request by triceo**
Makes sure all importers are reproducible.
Adjusts data sets so that all importers now reproducibly result in these specific file contents.

### CODE REVIEW

File: optaplanner-examples/src/main/java/org/optaplanner/examples/nurserostering/persistence/NurseRosteringImporter.java [hunk 1]
```diff
@@ -157,7 +157,7 @@ private void generateShiftDateList(NurseRoster nurseRoster,
             int maxDayIndex = Math.toIntExact(DAYS.between(startDate, endDate));
             int shiftDateSize = maxDayIndex + 1;
             List<ShiftDate> shiftDateList = new ArrayList<>(shiftDateSize);
-            shiftDateMap = new HashMap<>(shiftDateSize);
+            shiftDateMap = new LinkedHashMap<>(shiftDateSize);
```
Comment (rsynek) [inline optaplanner-examples/src/main/java/org/optaplanner/examples/nurserostering/persistence/NurseRosteringImporter.java:160]: Is the `LinkedHashMap` here to keep stable order of elements?

File: optaplanner-examples/src/test/java/org/optaplanner/examples/app/AllExamplesSolutionImporterTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package org.optaplanner.examples.app;
+
+import static java.util.stream.Collectors.*;
```
Comment (rsynek) [inline optaplanner-examples/src/test/java/org/optaplanner/examples/app/AllExamplesSolutionImporterTest.java:3]: star import

File: optaplanner-examples/src/test/java/org/optaplanner/examples/app/AllExamplesSolutionImporterTest.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import java.util.TreeMap;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.apache.commons.codec.digest.DigestUtils;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class AllExamplesSolutionImporterTest {
```
Comment (rsynek) [inline optaplanner-examples/src/test/java/org/optaplanner/examples/app/AllExamplesSolutionImporterTest.java:20]: How much time does this test take?

File: optaplanner-examples/src/main/java/org/optaplanner/examples/nurserostering/persistence/NurseRosteringImporter.java (hunk 1 above)
Comment (triceo) [reply optaplanner-examples/src/main/java/org/optaplanner/examples/nurserostering/persistence/NurseRosteringImporter.java:160]: Yes. That is what all the other LinkedHashMaps do, too.

File: optaplanner-examples/src/test/java/org/optaplanner/examples/app/AllExamplesSolutionImporterTest.java (hunk 3 above)
Comment (triceo) [reply optaplanner-examples/src/test/java/org/optaplanner/examples/app/AllExamplesSolutionImporterTest.java:20]: Not too long, surprisingly. A couple of seconds. Barely noticeable on my machine.

File: optaplanner-examples/src/test/java/org/optaplanner/examples/app/AllExamplesSolutionImporterTest.java (hunk 2 above)
Comment (triceo) [reply optaplanner-examples/src/test/java/org/optaplanner/examples/app/AllExamplesSolutionImporterTest.java:3]: Damn IDEA.

**Comment by triceo** [comment]
I'm scratching this experiment. The test is unreliable, the benefits will not outweigh the drawbacks.

**Pull request closed without merging**